torchrl/modules/models/models.py

```diff
@@ -1034,3 +1035,27 @@ def forward(
 
         input = self.mlp(input)
         return self._lstm(input, hidden0_in, hidden1_in)
+
+
+class GRUNet(nn.Module):
+    """
+    MLP -> GRU -> MLP
+
+    """
+
+    def __init__(
+        self, mlp_input_kwargs: Dict, gru_kwargs: Dict, mlp_output_kwargs: Dict
```

Comment:
it would be nice to have a `device` argument here, almost all torch modules now support instantiation on device